instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/Experimental.java

```diff
@@ -54,4 +61,27 @@ public static <REQUEST> void internalSetUrlTemplateExtractor(
           urlTemplateExtractorSetter) {
     Experimental.urlTemplateExtractorSetter = (BiConsumer) urlTemplateExtractorSetter;
   }
+
+  /**
+   * Add an {@link AttributesExtractor} to the given {@link InstrumenterBuilder} that provides
+   * attributes that are passed to the {@link OperationListener}s. This can be used to add
+   * attributes to the metrics without adding them to the span. To add attributes to the span use
+   * {@link InstrumenterBuilder#addAttributesExtractor(AttributesExtractor)}.
+   */
+  public static <REQUEST, RESPONSE> void addOperationAttributesExtractor(
```

Comment:
based on the general name (operation), I think I'd expect these attributes to be stamped onto both spans and metrics?

maybe OperationListenerMetrics? to be clear that these are only passed to OperationListeners

(if I read it right)